### DISCUSSION TITLE
Custom display of a widget

### DISCUSSION THREAD

**Discussion by MKececiInfo** [Azure/api-management-developer-portal#1633] [General]
Hi,

I have a requirement to display a widget to only the members of a particular group. Is this possible? If so any pointers would be greatly appreciated.

Thanks

**Comment by mikebudzynski** [comment]
Which widget specifically? Referencing #533

**Comment by MKececiInfo** [comment]
It's a custom widget that displays text/information.

We have 3 groups:
Devs
Customers
Partners

Based on the group of the subscriber, we want to display information in a text widget. On the API list & API details pages.

**Reply by mikebudzynski** [reply]
This will likely become possible once we complete two features that we're actively working on: extensibility of the managed developer portal with custom widgets and content access control. In the meantime, I recommend enforcing this condition in the data source for the widget.

**Reply by devagarwal007** [reply]
Is it live?
 Is it possible to display the product and operations using custom widget or update the existing one? Please help here.

**Reply by devagarwal007** [reply]
@mikebudzynski

**Comment by MKececiInfo** [comment]
Do these apply to the hosted version of the portal? We are currently using the hosted version. Is there a place where we can view & follow upcoming features that are planned & being worked on?

Thx

**Comment by bendixon-zelis** [comment]
@mikebudzynski We have the same requirement. We have a custom widget that we only want to display to users of certain groups similar to how access can be defined for built-in widgets. We're using the managed portal. Is there a way to enable the access control on a custom widget or is this feature still pending?
